examples/lang/erc20/src/lib.rs

```diff
@@ -86,162 +82,81 @@ contract! {
             total_supply
         }
 
-        /// Returns the balance of the given address.
+        /// Returns the balance of the given AccountId.
         pub(external) fn balance_of(&self, owner: AccountId) -> Balance {
-            let balance = *self.balances.get(&owner).unwrap_or(&0);
+            let balance = self.balance_of_or_zero(&owner);
             env.println(&format!("Erc20::balance_of(owner = {:?}) = {:?}", owner, balance));
```

Comment:
Deploy lost its `println` so either we keep all or we remove all.
What suits you better.

Reply:
What does this mean? Sorry, I do not understand.

Reply:
You removed the `env.println` call in `deploy` but not anywhere else. So what should we do here? Leave them? Remove all? I wouldn't keep a partial subset.

Reply:
I made it so that println is only used when calling getter functions. In this situation it is no so much a "debugging tool" as it is the only way to really interact with your contract meaningfully.

For all the other functions, I believe an event is emitted when that stuff happens, so it does not need a println